app/Models/Company/ProjectDecision.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace App\Models\Company;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
```

Comment:
You've missed some definition:
```suggestion
use Illuminate\Database\Eloquent\Relations\BelongsTo;
use Illuminate\Database\Eloquent\Relations\BelongsToMany;
```

Reply:
Fixed